ui/src/app/pages/admin/admin-user-access-sync.tsx

```diff
@@ -0,0 +1,269 @@
+import * as React from 'react';
```

Comment:
(I'm only here to comment on the UI)

What is your intention with this change?  Broad-Core is making a lot of changes to user-admin pages, so we should coordinate to avoid rework.

Reply:
Background slack discussion: https://pmi-engteam.slack.com/archives/CHRN2R51N/p1642716603017600
This is for the Jan 31th's RAS enforcement (That means we need to get this endpoint working in prod next week). The SAM endpoint might be problematic, and this is new solution. 
Then admin's can do a batch updates at one time, and since this is a small tool, so I put this component under admin user table. 
What would be your suggestion? 
1: The existing approach doesn't change the `admin-user.tsx` a lot. It is jus one line at the bottom of that page. I can leave it as it, and in Broad-core's admin-user pages, still put it at the bottom at that page. 
2:  I create a separate page (e.g. /admin-user-access). Then it won't change any of your work. 
wdyt?

Reply:
The UI change is actually a mistake commit, the working PR for the UI piece is here: 
https://github.com/all-of-us/workbench/pull/6206

Reply:
OK thanks!  I'll take a look over there.